recipes/pdfcombine/meta.yaml

```diff
@@ -0,0 +1,43 @@
+{% set name = "pdfcombine" %}
+{% set version = "0.3.2" %}
+{% set sha256 = "65d57f7024ce51c1f1c9123a1d71085288d5cec145d830b9a145d5c4e335f9ee" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/tdegeus/pdfcombine/archive/v{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  skip: True  # [py<35]
+
+requirements:
+  host:
+    - python
+  run:
+    - python
+    - docopt
+    - click
+    - ghostscript
+
+test:
+  commands:
+    - pdfcombine --help  # [unix]
+    - pdfcombine.py --help  # [win]
```

Comment:
Remove the `win` line, and remove `# [unix]` from the first one.

Reply:
The problem is that without the `*.py` extension, I'm getting:

```none
pdfcombine --help 
973'pdfcombine' is not recognized as an internal or external command,
974operable program or batch file.
```

Reply:
(Thanks for your help BTW!)

Reply:
Hmm, using no extension should work...

Reply:
It seems that adding the `*.py` extension is really the only way:

https://stackoverflow.com/a/7574585/2646505

Reply:
As far as I understand Windows (unless cygwin) has no notion of shebangs. I.e. without an extension it cannot decide with with interpreter to run the script. On unix systems this is exactly what the shebang is used for. 

I guess that the most consistent thing would be to use the extension also on unix, but I really don't want to go that far...